tests/constants.py

```diff
@@ -0,0 +1,7 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""This module defines Test-related constants"""
+
+TEST_SERVER_NAME = "server-name-configured.synapse.com"
+TEST_SERVER_NAME_CHANGED = "pebble-layer-1.synapse.com"
```

Comment:
Consider whether these could be part of a fixture

Reply:
We also use these values in some of the tests I think, If I put this value directly in a fixture I'd have to find a way to retrieve this value in the test functions. 

What do you think?